index.js

```diff
@@ -10,7 +10,8 @@ var noop = function() {}
 module.exports = function(db) {
   var feed = {}
   var lock = mutexify()
- 
+  var valueEncoding = db.options.valueEncoding || 'binary'
```

Comment:
lets add an `options` argument to module.exports and do `options.valueEncoding || db.options.valueEncoding || 'binary'`